### PR TITLE
6101 added fxcop support

### PR DESCRIPTION
**Pull request by SimonLLawrence**

### CODE REVIEW

File: src/NHSD.BuyingCatalogue.Ordering.Api/Properties/launchSettings.json [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    "NHSD.BuyingCatalogue.Ordering.Api": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "weatherforecast",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      },
+      "applicationUrl": "https://localhost:5001;http://localhost:5000"
+    },
+    "Docker": {
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Properties/launchSettings.json:29]: You dont need this

File: src/NHSD.BuyingCatalogue.Ordering.Api/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+      "launchBrowser": true,
+      "launchUrl": "weatherforecast",
+      "environmentVariables": {
+        "ASPNETCORE_ENVIRONMENT": "Development"
+      }
+    },
+    "NHSD.BuyingCatalogue.Ordering.Api": {
+      "commandName": "Project",
+      "launchBrowser": true,
+      "launchUrl": "weatherforecast",
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Properties/launchSettings.json:23]: Change the weatherforcast

File: src/NHSD.BuyingCatalogue.Ordering.Api/Properties/launchSettings.json [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+      "applicationUrl": "http://localhost:53915",
+      "sslPort": 44355
+    }
+  },
+  "$schema": "http://json.schemastore.org/launchsettings.json",
+  "profiles": {
+    "IIS Express": {
+      "commandName": "IISExpress",
+      "launchBrowser": true,
+      "launchUrl": "weatherforecast",
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Ordering.Api/Properties/launchSettings.json:15]: Same up here

File: src/NHSD.BuyingCatalogue.Ordering.Api/NHSD.BuyingCatalogue.Ordering.Api.csproj [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.9.8">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+  </ItemGroup>
+
+  <ItemGroup>
+    <Folder Include="Controllers\" />
+  </ItemGroup>
+
```
Comment (RuthBerry96) [inline src/NHSD.BuyingCatalogue.Ordering.Api/NHSD.BuyingCatalogue.Ordering.Api.csproj:25]: Extra space

**Comment by SimonLLawrence** [comment]
Merged

**Pull request merged**